Source/DafnyCore/Compilers/Java/Compiler-java.cs

```diff
@@ -2852,7 +2868,7 @@ protected class ClassWriter : IClassWriter {
     protected override void EmitFooter(Program program, ConcreteSyntaxTree wr) {
       // Emit tuples
       foreach (int i in tuples) {
-        if (i == 2 || i == 3) {
+        if (i <= 20) {
           continue; // Tuple2 and Tuple3 already exist in DafnyRuntime.jar, so don't remake these files.
```

Comment:
This comment seems obsolete w.r.t. the updated condition, though that's probably not worth holding this up for.